src/validation_workflow/validation.py

```diff
@@ -38,6 +41,17 @@ def copy_artifact(self, filepath: str, tempdir_path: str) -> bool:
         else:
             raise Exception("Provided path for local artifacts does not exist")
 
+    def test_security_plugin(self, work_dir: str) -> bool:
+        (_, path, _) = execute(f'find {work_dir} -type f -iname \'opensearch-plugin\'', ".", True, False)
+        if (path):
+            (_, list_plugins, _) = execute("./opensearch-plugin list", path.replace("opensearch-plugin", "").rstrip("\n"), True, False)
```

Comment:
this should be `./bin.opensearch-plugin list` right?

Reply:
Yeah Sayali, but `find` retreives the filepath till `../bin/opensearch-plugin` , Just removed the `opensearch-plugin`, so now it should be in bin folder. The command executes perfectely.

Reply:
Can you explain what you are trying to do here?
Trying to find opensearch-plugin location? Or trying to find the plugin folder, because your line 50 `            raise Exception("Couldn't fetch the path to plugin folder")` indication you are looking for the plugin folder.

Reply:
After talking to @Divyaasm it is clear that she wants to find opensearch-plugin utilility then use it to find security plugin.

The `find` method is not efficient and is not the correct way to get the location of `opensearch-plugin`, especially that windows is using a different variation of `find` that does not run with gnu syntax.

Suggesting following https://github.com/opensearch-project/opensearch-build/blob/main/src/test_workflow/integ_test/distribution_rpm.py or similar to implement abstract method to find the correct opensearch-plugin path, or even security plugin folder directly in child class.

Thanks.

Reply:
Checking $OPENSEARCH_HOME/plugins/opensearch-security might be an easier way so you do not need to execute any command.